### ISSUE TITLE
Is this gem still maintained?

### ISSUE THREAD

**Issue by lukasztackowiak** [podigee/device_detector#97]
I see that regexes from original repository has been imported last time at 2022-02-17 (8 months ago at a time when I'm writing this) so I'd like to ask if this gem is still maintained by author and if updates can be expected in future?

**Comment by benzimmer** [comment]
@lukasztackowiak Yes, the gem is still maintained and we will update the regexes as soon as possible.

**Comment by girardof** [comment]
@lukasztackowiak A [new version](https://rubygems.org/gems/device_detector/versions/1.1.0) of device_detector was just released, all the regexes are up to date with [Matomo's repository](https://github.com/matomo-org/device-detector).
